qiime2/sdk/usage.py

```diff
@@ -922,6 +922,146 @@ def init_format(self, name: str,
         """
         return self._usage_variable(name, factory, 'format')
 
+    def _replace_url_epoch(self, url):
+        # Obtaining epoch modeled on qiime2.metadata.io.MetadataFileError
+        import qiime2
```

Comment:
```suggestion
```
`qiime2` is being imported at the top of this file, so it shouldn't be needed as an additional import here.